packages/build/src/core/main.js

```diff
@@ -233,9 +234,20 @@ const initAndRunBuild = async function({
   logs,
   timers,
   testOpts,
+  buildbotServerSocket,
 }) {
   const constants = await getConstants({ configPath, buildDir, functionsDistDir, netlifyConfig, siteInfo, mode })
 
+  var buildbotClient
+  if (buildbotServerSocket) {
+    try {
+      buildbotClient = await startBuildbotClient(buildbotServerSocket)
+    } catch (error) {
```

Comment:
Error handling is already done inside `startBuildbotClient()`, so this might be skipped?